CONTRIBUTING.rst

```diff
@@ -1,177 +1,228 @@
-- *Reference* - Referential content provides detailed information that people need while they're actively using Aiven's products.
-- *Tutorial* - Tutorials help people learn about products, and solve real world problems by guiding them through the entire workflow to complete a task.
+The instructions for setting up local development are in the ``README``.
 
-.. tip::
-    Use the templates included below for further guidance. 
+0. We heartily recommend you open an issue so that we can either help with outlines, or know that three people are working on the same thing.
 
-To learn more about documentation content types, see the `Diátaxis Framework <https://diataxis.fr/>`_.
+1. Make your changes on a branch. We recommend a maximum of three articles, and don't review large changesets without prior discussion. File names are ``kebab-case``, with hyphens, such as ``connect-with-pgadmin.rst``. Files should also be added to the table of contents file ``_toc.yml`` so they appear in the right location in the navigation.
 
+2. Open a pull request, use the template so we know what it is and why it's a great addition. If you need to publish this quickly for a reason, mention that as well.
 
-**What is the structure of DevPortal?**
+3. We have an automated build process that checks your work against our in-house styles, gets product names with the right capitalization, and checks links. Please don't be alarmed if your build fails! View the output of the failed jobs to find out what happened, or we can advise when we review. You can run these locally if you like: ``make spell`` and ``make linkcheck``.
 
-You can see the structure in the left-hand navigation menu. 
-It is driven by a plugin called `Sphinx external TOC <https://sphinx-external-toc.readthedocs.io/en/latest/intro.html>`_. 
+.. tip::
 
-You can find the current structure in ``_toc.yml``.
+    Don't add commands to the accepted words list, these should always be in ``literal`` markup and ignored by the spell checker. Literal markup can't be used inside link text, so reword your link and sentence to avoid this.
 
-If you have an article about any of the Aiven tools (API, CLI, console, Terraform provider, Kubernetes operator), place it in the **Aiven tools** section.
-If you have an article about any specific Aiven product (M3, Kafka, Grafana, etc.), nest it in the product-specific section. 
-If you have an article with general reference information, place it in the **Resources** section. 
+4. Every pull request generates a preview link, that will be added to your pull request as a comment. This link is publicly available and does not change during the lifetime of your pull request. Feel free to share it with others who might want a preview of your changes.
 
+5. Our docs team will review your pull request, and you will receive some feedback. It's expected that there will be 2 or 3 rounds of revisions, but if you'd rather we just made changes ourselves, let us know.
```

Comment:
I was thinking the same - we should have an eye for number of revisions, so that those who submit don't feel like it is impossible to make it right... But with the new guidelines this should become easier!